### ISSUE TITLE
Hitting the 'Return' key when filling out an inline create takes you to a 405 error page. 

### ISSUE THREAD

**Issue by selected-pixel-jameson** [Laravel-Backpack/CRUD#3172]
# Bug report

### What I did

Created a Relationship field type and set it up so that I could create inline content. 

### What I expected to happen

I clicked the +Add option. Enter my values and HIT THE RETURN key on my keyboard. 

### What happened

Regardless of what I entered in the form I'm taken to a 405 Error page.

ERROR
405
Method not allowed.
The POST method is not supported for this route. Supported methods: GET, HEAD.

### What I've already tried to fix it

Nothing.

### PHP VERSION:
PHP 7.3.15-3+ubuntu18.04.1+deb.sury.org+1 (cli) (built: Feb 23 2020 07:23:33) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.15, Copyright (c) 1998-2018 Zend Technologies
    with Zend OPcache v7.3.15-3+ubuntu18.04.1+deb.sury.org+1, Copyright (c) 1999-2018, by Zend Technologies

### LARAVEL VERSION:
v6.18.19@69321afec31f4a908112e5dc8995fc91024fd971

### BACKPACK VERSION:
4.1.8@393a054490c97721186deb9faed20c6ed70ef6ef

**Comment by pxpm** [comment]
Hello @selected-pixel-jameson 

Does the same happen if you click the button?

My guess is that the enter key is bind to main parent form and not the inline create
 Could you confirm?

Best,
Pedro

**Comment by selected-pixel-jameson** [comment]
No. If I click the button it works as it should.

Jameson W Parker

**Comment by pxpm** [comment]
@selected-pixel-jameson taking a better look at it, it should be fixed by now.

It was reported in #2845 

I fixed it in: #2855 

Make sure you have the the correct version for file. 

You should have something like this in ine 454 of fetch_and_create:

```php
 $modal.on('shown.bs.modal', function (e) {
        $modal.on('keyup',  function (e) {
        if($modal.is(':visible')) {
            var key = e.which;
                if (key == 13) { //This is an ENTER
                e.preventDefault();
                $modalSaveButton.click();
            }
        }
        return false;
    });
    });
```

There is also changes in `inline_create_modal`. Follow the commit I showed you for all the changes and make sure to apply them if you are overriding any of the files.

I am going to close this as I think this is already solved.

Could you confirm ? 

Best,
Pedro

**Comment by selected-pixel-jameson** [comment]
Updated and the problem is resolved. Thank you.
